workers/unity/Packages/com.improbable.gdk.tools/.DownloadCoreSdk/Program.cs

```diff
@@ -29,7 +29,13 @@ private static void Main(string[] args)
                 new Package(tempPath, "worker_sdk", "c-dynamic-x86_64-msvc_mt-win32", $"{nativeDependenciesPath}/Windows/x86_64", new List<string> {"include", "worker.lib"}),
                 new Package(tempPath, "worker_sdk", "c-dynamic-x86_64-gcc_libstdcpp-linux", $"{nativeDependenciesPath}/Linux/x86_64", new List<string> {"include"}),
                 new Package(tempPath, "worker_sdk", "c-bundle-x86_64-clang_libcpp-macos", $"{nativeDependenciesPath}/OSX", new List<string> {"include"}),
-                new Package(tempPath, "worker_sdk", "csharp_core", $"{managedDependenciesPath}/OSX"),
+                new Package(tempPath, "worker_sdk", "c-static-fullylinked-arm-clang_libcpp-ios", $"{nativeDependenciesPath}/iOS/arm", new List<string> {"include", "libworker_static_fullylinked.a.pic", "worker_static_fullylinked.lib"}),
+                new Package(tempPath, "worker_sdk", "c-static-fullylinked-x86_64-clang_libcpp-ios", $"{nativeDependenciesPath}/iOS/x86_64", new List<string> {"include", "libworker_static_fullylinked.a.pic", "worker_static_fullylinked.lib"}),
```

Comment:
should we move it to `Plugins/iOS` so that Unity takes care of that? or do we rely on the plugin installer for that?

Reply:
I don't mind changing the structure overall.
We still will have to add custom plugin compatibility logic at least to exclude WorkerSDK dll from iOS and add architecture data to Android.
So, my approach was to keep structure as is and just mark compatibility in script, but I don't have strong opinion on this

Reply:
if we still have to do this, we can leave it in here I guess